libuavcan_drivers/linux/include/uavcan_linux/socketcan.hpp

```diff
@@ -148,6 +148,9 @@ class SocketCanIface : public uavcan::ICanIface
     std::queue<RxItem> rx_queue_;                                   // TODO: Use pool allocator
     std::unordered_multiset<std::uint32_t> pending_loopback_ids_;   // TODO: Use pool allocator
 
+    std::vector<uavcan::CanFilterConfig> hw_filters_container_;
+    bool filters_configured_ = false;
```

Comment:
This variable can be replaced with a check like `!hw_filters_container_.empty()`. I suggest to remove it. It's always better to have as few states as possible.